run_gmsh/demo_create_gmsh_mesh.py

```diff
@@ -35,9 +27,31 @@
 mesh_output_dir = "./"
 
 
+# Extract the points, mesh sizes and name labels from the dictionary
+input_points_list = []
+mesh_size_list = []
+name_label_list = []
+for value in input_dict.values():
+    input_points_list.append(list(value[1]))  # Extract the points
+    if value[0] <= 0:  # Check if mesh size is negative
+        warnings.warn('Warning Message: Mesh size cannot be negative!')
```

Comment:
when the user fills in a negative number, the mesh size is set to the default value (chosen by gmsh). Better to say that in the warning. And not changing it to 1